src/wallet/api/wallet.cpp

```diff
@@ -1952,6 +1952,10 @@ bool WalletImpl::unblackballOutput(const std::string &pubkey)
     return true;
 }
 
+string WalletImpl::setRingDatabase(const std::string &filename) {
+    m_wallet->set_ring_database(filename);
+}
```

Comment:
This doesn not return the claimed string.

Reply:
It's not meant to, it's a setter, it's to allow setting the ring db dir externally, exposing the function to the wallet api

Reply:
oh nevermind I see it now, derp